src/components/notifications/static/static.story.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import withReadme from 'storybook-readme/with-readme';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import Section from '../../../../.storybook/decorators/section';
+import Readme from './README.md';
+import Static from './static';
+
+storiesOf('Notifications', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withReadme(Readme))
+  .add('Static', () => (
+    <Section>
+      <Static type="warning">
```

Comment:
Most of our stories are set up in a way where the component is rendered only once. The different states can be interacted with using the knobs. This helps designers figure out which knobs (props) exist and to communicate better with developers.

Can you change the component to be rendered once only and to use a knob for the `type` attribute?

I know that it's not nice to see all our notifications at once, but that's not what we're aiming for in the storybook documentation. The goal would be to have a dedicated documentation website which shows off the components better later on.